trigger/payloads.go

```diff
@@ -438,7 +438,7 @@ func getFailedTestsFromTemplate(t task.Task) ([]task.TestResult, error) {
 }
 
 func taskLink(uiBase string, taskID string, execution int) string {
-	return fmt.Sprintf("%s/task/%s/%d", uiBase, url.PathEscape(taskID), execution)
+	return fmt.Sprintf("%s/task/%s/%d/?redirect_spruce_users=true", uiBase, url.PathEscape(taskID), execution)
```

Comment:
The image you shared has a malformed link. (Two forward slashes after mongodb.com) Can you update the link to remove the extra forward slash. 
![image](https://user-images.githubusercontent.com/4605522/140163075-41cc4297-994a-4e31-b259-983ad923b311.png)

Reply:
We have some logic in gimlet that parses out the double forward slash but we shouldn't be generating links like that anyways.  https://github.com/evergreen-ci/gimlet/pull/97

Reply:
also shouldn't put a slash before ? for correct syntax https://help.marketruler.com/wiki/What_is_the_correct_syntax_for_query_strings%3F

Reply:
done